### PR TITLE
integrating nunavut

### PR DESCRIPTION
**Pull request by thirtytwobits**
I am so ... sick ... of ... cmake!

### CODE REVIEW

File: docs/examples/example_01_hello_world.cpp [hunk 1]
```diff
@@ -14,11 +14,28 @@
 
 #include <gtest/gtest.h>
 
+#include "cetl/pf17/sys/memory_resource.hpp"
+#include "cetl/pf17/byte.hpp"
+#include "cetl/variable_length_array.hpp"
+
+#include "example/foo_1_0.hpp"
```
Comment (lydia-at-amazon) [inline docs/examples/example_01_hello_world.cpp:21]: Did you also have an example of including a regulated data type?

**Review by lydia-at-amazon — APPROVED** [review]

**Comment by lydia-at-amazon** [comment]
Should we expect the checks to pass?

**Review by emrainey — APPROVED** [review]
Would like to see some usage w/ the common standard messages (like one or two) but that doesn't have to be here.

File: docs/examples/example_01_hello_world.cpp (hunk 1 above)
Comment (thirtytwobits) [reply docs/examples/example_01_hello_world.cpp:21]: Wow. It's like you two don't trust me ;-) Yah. Will add...

**Review by emrainey — APPROVED** [review]

**Pull request merged**